tests/integration/backward_compatible/serialization/compact_compatibility/compact_compatibility_test.py

```diff
@@ -1402,6 +1402,23 @@ def assertion():
     def test_publish(self):
         self.topic.publish(OUTER_COMPACT_INSTANCE)
 
+    def test_publish_all(self):
+        messages = []
```

Comment:
there should be a `skip_if_client_version_older_than(self, "5.2")` call here as well

Reply:
I thought the `CompactCompatibilityBase` handles this check, but I guess it only checks for `5.1`. Added `5.2` check for `publish_all` at https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/b91a6e5018e63f4a14938e3680d9762d4586863e.